### PR TITLE
JBIDE-26890: fix nodeclipse repository

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stéphane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

File: devstudio/com.jboss.devstudio.central.discovery/plugin.xml [hunk 1]
```diff
@@ -104,7 +104,7 @@ reasonable, reporting issues to these providers as required.</description>
             license="EPL, ASL, others (Free)"
             name="Nodeclipse for node.js"
             provider="Nodeclipse"
-            siteUrl="https://nodeclipse.github.io/updates/">
+            siteUrl="http://dl.bintray.com/nodeclipse/nodeclipse/1.0.2f/">
```
Comment (jeffmaury) [inline devstudio/com.jboss.devstudio.central.discovery/plugin.xml]: Switch to https

File: jbosstools/org.jboss.tools.central.discovery/plugin.xml [hunk 2]
```diff
@@ -71,7 +71,7 @@ Supported by Red Hat.</description>
             license="EPL, ASL, others (Free)"
             name="Nodeclipse for node.js"
             provider="Nodeclipse"
-            siteUrl="https://nodeclipse.github.io/updates/">
+            siteUrl="http://dl.bintray.com/nodeclipse/nodeclipse/1.0.2f/">
```
Comment (jeffmaury) [inline jbosstools/org.jboss.tools.central.discovery/plugin.xml]: Switch to https

**Comment by sbouchet** [comment]
changes applied.

File: jbosstools/org.jboss.tools.central.discovery/plugin.xml (hunk 2 above)
Comment (sbouchet) [reply jbosstools/org.jboss.tools.central.discovery/plugin.xml]: done.

**Pull request merged**